src/utils/config/ethereum.ts

```diff
@@ -42,7 +42,7 @@ export const mainnet: NetworkConfig = {
     toAddressExplorerUrl: (token: string): string =>
         `https://etherscan.io/address/${token}`,
     toTxExplorerUrl: (tx: string): string => `https://etherscan.io/tx/${tx}`,
-    subgraphUrl: `https://gateway.thegraph.com/api/${env.theGraphKey}/subgraphs/id/5xMSe3wTNLgFQqsAc5SCVVwT4MiRb5AogJCuSN9PjzXF`,
+    subgraphUrl: `https://api.thegraph.com/subgraphs/name/salazarguille/yearn-vaults-v2-subgraph-mainnet`,
```

Comment:
Restore `sugraphUrl` we used before so that we can merge for production.
The api you used is only for development.